js/stringProcessing/getSentences.js

```diff
@@ -10,15 +10,18 @@ var getSubheadings = require( "./getSubheadings.js" ).getSubheadings;
 // All characters that indicate a sentence delimiter.
 var sentenceDelimiters = ".?!:;";
 
+// If a sentence delimiter is followed by one of these characters, it is a valid ending.
+var sentenceEndingRegex = /[\s<\)\]\"\']/;
+
 /**
- * Checks if the period is followed with a whitespace. If not, it is no ending of a sentence.
+ * Checks if the period is followed with a character that is a valid sentence ending. If not, it is no ending of a sentence.
```

Comment:
The last sentence is a bit weird. Perhaps change this to `If not, it is not an ending of a sentence.`.